### PR TITLE
TRT-974: Log error but continue with VariantTestReport

### PR DESCRIPTION
**Pull request by neisw**

### CODE REVIEW

File: pkg/api/install.go [hunk 1]
```diff
@@ -107,9 +107,11 @@ func VariantTestsReport(dbc *db.DB, release string, reportType v1.ReportType,
 	for testName := range tests {
 		allReport, err := query.TestReportExcludeVariants(dbc, release, testName, excludedVariants)
 		if err != nil {
-			return variantColumns, tests, err
+			// log the error and keep going
+			log.WithError(err).Errorf("Failed to query test report for: %s", testName)
+		} else {
+			tests[testName]["All"] = allReport
```
Comment (stbenjam) [inline pkg/api/install.go:113]: What does allReport contain if it returns an error?

Comment (neisw) [reply pkg/api/install.go:113]: Looks empty in the debugger and based on https://github.com/openshift/sippy/blob/c343a00e15c79c9ee6ce27aed51d3eb7a9147869/pkg/db/query/test_queries.go#L151

**Pull request merged**